nemo/collections/llm/gpt/model/base.py

```diff
@@ -204,6 +207,9 @@ class GPTConfig5B(GPTConfig):
     ffn_hidden_size: int = 16384
     num_attention_heads: int = 32
 
+    bias_activation_fusion: bool = True
+    bias_dropout_add_fusion: bool = True
+
```

Comment:
Can these be removed since you already set these in base.py?

Reply:
This is in base.py. Do you mean some other file where this is set?

Reply:
Oh my mistake, could you set these in `GPTConfig`  instead?

Reply:
Ok. I'll set them here. But, also we have to set them to False in `NemotronConfig` then.

Reply:
Do they cause errors if you set them in Nemotron?

Reply:
Haven't tried. It is set to False for all nemotron models in Launcher.

Reply:
This breaks other model tests in CI. Reverting to model specific fusion params.